project-base/app/patch/overblog-validator.patch

```diff
@@ -0,0 +1,11 @@
+--- ./vendor/overblog/graphql-bundle/src/Validator/InputValidator.php
++++ ./vendor/overblog/graphql-bundle/src/Validator/InputValidator.php
+@@ -118,7 +118,7 @@
+             // @phpstan-ignore-next-line (only for Symfony 4.4)
+             $builder
+                 ->setTranslator($this->defaultTranslator)
+-                ->setTranslationDomain('validators');
++                ->setTranslationDomain('customerValidators');
+         }
```

Comment:
We are not able to use the constant here, or?

Reply:
As it's a code from vendor, we would have to import the class, or use FQCN. I'm not sure if it's better